packages/Card/src/components/Title/Title.js

```diff
@@ -5,17 +5,22 @@ import * as sc from "./Title.styles";
 const propTypes = {
   /** Body content of the Title. */
   children: PropTypes.node,
+  /** Semantic level of Heading for the title */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
 };
 
 const defaultProps = {
   children: null,
+  level: 1,
 };
 
-const Title = props => {
-  const { children } = props;
+const safeValue = n => (n === undefined || n < 1 || Number.isNaN(n) ? 6 : Math.min(n, 6));
 
+const Title = props => {
+  const { children, level } = props;
+  const safeLevel = safeValue(level);
   return (
-    <sc.titleStyles data-pka-anchor="card.title" {...props}>
+    <sc.titleStyles data-pka-anchor="card.title" as={`h${safeLevel}`} {...props}>
```

Comment:
If `<sc.titleStyles>` was a styled `<Heading>` from Paprika, then you could just pass the `level` prop right through, or even pass along all `moreProps`, and avoid the `safeValue` logic above since its already in that component. 

In fact, the styling of `<sc.titleStyles>` (or just `<sc.Title>`) could be reduced too if the default `displayLevel` prop was provided with the appropriate value..